airflow/contrib/operators/bigquery_to_gcs.py

```diff
@@ -72,18 +73,18 @@ class BigQueryToCloudStorageOperator(BaseOperator):
 
     @apply_defaults
     def __init__(self,  # pylint: disable=too-many-arguments
-                 source_project_dataset_table,
-                 destination_cloud_storage_uris,
-                 compression='NONE',
-                 export_format='CSV',
-                 field_delimiter=',',
-                 print_header=True,
-                 gcp_conn_id='google_cloud_default',
-                 bigquery_conn_id=None,
-                 delegate_to=None,
-                 labels=None,
+                 source_project_dataset_table: str,
+                 destination_cloud_storage_uris: List,
```

Comment:
Do we know what type have elements of the list?

Reply:
I've checked documentation and it should be list of strings. I will update it.